compiler/rustc_target/src/spec/targets/armv6_unknown_freebsd.rs

```diff
@@ -8,8 +8,7 @@ pub fn target() -> Target {
         arch: "arm".into(),
         options: TargetOptions {
             abi: "eabihf".into(),
-            // FIXME: change env to "gnu" when cfg_target_abi becomes stable
-            env: "gnueabihf".into(),
+            env: "gnu".into(),
```

Comment:
Are there cfgs in std that rely on this? Or is this only used in third party code? It also makes sense to ping the target maintainers (if they exist) here to notify them.

Reply:
None in std. I did a ripgrep on all of library/ and the only thing that came up was stdarch where it was only used as part of the full target name.